stories/guides/fozzie/fozzie-migration-guide.stories.mdx

```diff
@@ -143,3 +143,20 @@ module.exports = {
 }
 </style>
 ```
+
+<a name="gulp-migration"></a><h2>If you're coming from Gulp</h2>
```

Comment:
Would this be a more accurate heading?
```suggestion
<a name="gulp-migration"></a><h2>If you're not using Webpack</h2>
```

Reply:
I think for this one it should stay as Gulp as we want users who are currently on these tools to have a section specific to them. There's two teams using Gulp at the moment.